src/alert-prelude.c

```diff
@@ -207,12 +207,18 @@ static int EventToImpact(const PacketAlert *pa, const Packet *p, idmef_alert_t *
     SCEnter();
 
     ret = idmef_alert_new_assessment(alert, &assessment);
-    if (unlikely(ret < 0))
+    if (unlikely(ret < 0)) {
+        SCLogDebug("%s: error creating assessment: %s.",
```

Comment:
It is an error, creating an assessment should always be possible. But as you said, if it happened one time, it could happen many time, so I use debug instead of Error.